### PR TITLE
AP-3297 sdk

### PR DESCRIPTION
**Pull request by CleanBread**
i suppose we should write more in overview section but idk what exactly

### CODE REVIEW

File: pages/sdk/data-hooks/useBets.mdx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+#### Usage
+
+```ts
+import { useBets } from '@azuro-org/data'
+
+const { loading, error, data: bets } = useBets(props)
+```
+
+<Callout type="warning">
+  The hook won't be called until `bettor` value is nullish.
```
Comment (on47sky) [inline pages/sdk/data-hooks/useBets.mdx:14]: wrong meaning, should be
"until `bettor` value is defined"
or
"while `bettor` value is nullish"

File: pages/sdk/data-hooks/useBetsCache.mdx [hunk 2]
```diff
@@ -0,0 +1,82 @@
+Usied for update existing bet cache or add new bet to.
```
Comment (on47sky) [inline pages/sdk/data-hooks/useBetsCache.mdx:1]: typo, Used

File: pages/sdk/data-hooks/useGames.mdx [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+```
+
+#### Props
+
+```ts
+{
+  filter?: {
+    limit?: number // limit the number of rows returned from a query
+    offset?: number // omit a specified number of rows before the beginning of the result set
+    sportName?: string // returns games from specific sport
```
Comment (on47sky) [inline pages/sdk/data-hooks/useGames.mdx:26]: wrong param, should be `sportSlug` (the same typo in sdk readme)

File: pages/sdk/installation.mdx [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+```
+
+## Peer Dependencies
+
+```
+@apollo/client^3.8.0-beta.3 
+@apollo/experimental-nextjs-app-support@^0.2.1
+graphql@^16.6.0
+react@^18.2.0
+view^0.3.50
```
Comment (on47sky) [inline pages/sdk/installation.mdx:14]: viem

File: pages/sdk/providers/apollo.mdx [hunk 5]
```diff
@@ -0,0 +1,40 @@
+import { Callout } from 'components'
+
+Used to settup Apollo provider for your React or Next.js application.
```
Comment (on47sky) [inline pages/sdk/providers/apollo.mdx:3]: typo, "setup"

File: pages/sdk/data-hooks/useBets.mdx (hunk 1 above)
Comment (CleanBread) [reply pages/sdk/data-hooks/useBets.mdx]: cff9b0a6319353850ff3a3a5808d675a1f955ddc

File: pages/sdk/data-hooks/useBetsCache.mdx (hunk 2 above)
Comment (CleanBread) [reply pages/sdk/data-hooks/useBetsCache.mdx]: 17006da3ee23e6b3a29900c5cf6fcd64398d1f77

File: pages/sdk/data-hooks/useGames.mdx (hunk 3 above)
Comment (CleanBread) [reply pages/sdk/data-hooks/useGames.mdx]: 5836f805b245b40d658f9236f6dcc69334290cb2

File: pages/sdk/installation.mdx (hunk 4 above)
Comment (CleanBread) [reply pages/sdk/installation.mdx]: 485422602b802ba3c323aa22f544919cf6f22ff3

File: pages/sdk/providers/apollo.mdx (hunk 5 above)
Comment (CleanBread) [reply pages/sdk/providers/apollo.mdx]: 9c41231769ce7877df2af70a06dc68743b819c34

**Pull request merged**